manifests/dcgm/telegraf.pp

```diff
@@ -22,106 +14,39 @@
     $ensure_parm = 'absent'
   }
 
+  file { '/etc/telegraf/telegraf.d/dcgmd.conf':
+    ensure  => 'file',
+    content => file("${module_name}/dcgmd.conf"),
+    mode    => '0640',
+    owner   => 'root',
+    group   => 'telegraf',
+    notify  => Service['telegraf'],
+  }
+
+  file { '/etc/telegraf/scripts/dcgm':
+    ensure  => 'directory',
+    mode    => '0650',
+    owner   => 'root',
+    group   => 'telegraf',
+  }
+
+  file { '/etc/telegraf/scripts/dcgm/dcgmi_stats.sh':
+    ensure  => 'file',
```

Comment:
You'll want to change this line to read:
ensure => $ensure_parm,

Since we'll for sure want to cleanup `/etc/telegraf/telegraf.d/dcgmd.conf` we might was well cleanup the script too.

Reply:
Done, I've made change on my local machine.  Will wait to commit once I've gathered any other input then will rebase/squash commits all in one shot